chainservice/builder.go

```diff
@@ -343,7 +343,7 @@ func (builder *Builder) buildBlockDAO(forTest bool) error {
 		}
 		dbConfig := cfg.DB
 		if bsPath := cfg.Chain.BlobStoreDBPath; len(bsPath) > 0 {
-			blocksPerHour := time.Hour / cfg.DardanellesUpgrade.BlockInterval
+			blocksPerHour := time.Hour / cfg.WakeUpgrade.BlockInterval
```

Comment:
The time for blob storage can be calculated immediately in 3s block interval, which means that some additional blocks are stored without needing to wait for the hard fork to take effect.

Reply:
would this cause inconsistent issue? like some node already upgraded so think certain blocks received from blocksync should carry blob data, while (the sending) node that haven't upgraded still use 5 seconds and think the block should not carry blob, or vice versa? think about this situation thoroughly

Reply:
blob validation is not according to this, using `MinBlocksForBlobRetention` instead, which does not be changed